compiler/damlc/tests/daml-test-files/AuthFailure.daml

```diff
@@ -13,13 +15,13 @@ template TheContract1
   where
     signatory s
 
--- @ERROR failed due to a missing authorization from 'Ivy-t1'
-t1_CreateMissingAuthorization = scenario do
-  mach <- getParty "Mach-t1"
-  ivy <- getParty "Ivy-t1"
-  submit mach $ do
-    create (TheContract1 with s = ivy)
-    abort "t1 finished"
+-- @ERROR range=20:1-20:30; failed due to a missing authorization from 'Ivy'
+t1_CreateMissingAuthorization : Script ()
+t1_CreateMissingAuthorization = script do
+  mach <- allocateParty "Mach"
+  ivy <- allocateParty "Ivy"
+  mach `submit` createCmd (TheContract1 with s = ivy)
+  abort "t1 finished"
```

Comment:
(very minor remark) 

Originally the `abort` is always done inside the ledger (inside the Update monad), here you make it outside. 
In fact I prefer it like that. For the sake of consistency I will make it consistently outstide.